apps/condo/domains/billing/schema/RegisterBillingReceiptsService.js

```diff
@@ -282,7 +229,7 @@ const RegisterBillingReceiptsService = new GQLCustomSchema('RegisterBillingRecei
             access: true,
             type: `input RegisterBillingReceiptAddressMetaInput {
                 globalId: String
-                addressKey: String
```

Comment:
why is it so?

Reply:
in registry integration we already normalise addresses before sending them to unified mutation. So we can use addressKey to avoid additional requests on receipts creation. But for now - address service do not support such flow